src/Data/List/Properties.agda

```diff
@@ -764,84 +764,152 @@ module _ {P : A → Set p} (P? : Decidable P) where
   ...  | no ¬Px = P.cong (Prod.map id (x ∷_)) (partition-defn xs)
 
 ------------------------------------------------------------------------
--- reverse
+-- reverse is an involution in the list monoid
 
 module _ {a} {A : Set a} where
```

Comment:
I know this isn't your code, but might as well remove this anonymous module while we're here :smile:

Reply:
This module allows me to use `open` without polluting the namespace for code further done.  Thus, I think it is good to keep it!

Reply:
Fair enough :+1: